### PR TITLE
[JENKINS-65398] Terminology update

### PR DESCRIPTION
**Pull request by twasyl**
Terminology update as part of [JENKINS-65398](https://issues.jenkins.io/browse/JENKINS-65398).

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,11 +54,11 @@ be changed later. If you skip this step, you will be very confused and not get v
 ## Executors used by a multi-configuration project
 Reference: [Re: Will a multi-configuration / matrix job always use up one executor on the master?](http://groups.google.com/group/jenkinsci-users/msg/eb809fb06759d861)
 
-A matrix build project (that uses the Agent axis) will use one additional executor ("master executor"?)
+A matrix build project (that uses the Agent axis) will use one additional executor ("controller executor"?)
```
Comment (Vlatombe) [inline README.md:57]:
```suggestion
A matrix build project (that uses the Agent axis) will use one additional executor
```

File: README.md [hunk 2]
```diff
@@ -54,11 +54,11 @@ be changed later. If you skip this step, you will be very confused and not get v
...
 on a random node to coordinate the executions on the nodes defined by the Agent axis.
 This executor is added as a temporary executor (also called as "flyweight task") to the node and does not
 use up a configured executor slot on this node (this behavior is controlled by hudson.model.Hudson.flyweightSupport).
-Only this "master executor" or "flyweight task" will be affected by the "Restrict where this project can be run"
+Only this "controller executor" or "flyweight task" will be affected by the "Restrict where this project can be run"
```
Comment (Vlatombe) [inline README.md:61]:
```suggestion
Only this "flyweight task" will be affected by the "Restrict where this project can be run"
```

Comment (Vlatombe) [reply README.md:61]: I think it is clearer without these which don't refer to anything usual in the Jenkins terminology.

File: README.md [hunk 3]
```diff
@@ -52,13 +52,13 @@ be changed later. If you skip this step, you will be very confused and not get v
  You have to drill into each configuration to see the history and console logs.
 
 ## Executors used by a multi-configuration project
-Reference: [Re: Will a multi-configuration / matrix job always use up one executor on the master?](http://groups.google.com/group/jenkinsci-users/msg/eb809fb06759d861)
+Reference: [Re: Will a multi-configuration / matrix job always use up one executor on the build-in node?](http://groups.google.com/group/jenkinsci-users/msg/eb809fb06759d861)
```
Comment (amuniz) [inline README.md:55]:
```suggestion
Reference: [Re: Will a multi-configuration / matrix job always use up one executor on the built-in node?](http://groups.google.com/group/jenkinsci-users/msg/eb809fb06759d861)
```

**Review by amuniz — APPROVED** [review]

File: src/main/java/hudson/matrix/FilterScript.java [hunk 4]
```diff
@@ -137,7 +137,7 @@ public boolean apply(AxisList axes, Combination combination, Binding binding) {
         }
     };
 
-    // TODO JENKINS-25804: harmless generic methods like this should be whitelisted in script-security
+    // TODO JENKINS-25804: harmless generic methods like this should be allowlisted in script-security
```
Comment (aHenryJard) [inline src/main/java/hudson/matrix/FilterScript.java:140]:
```suggestion
    // TODO JENKINS-25804: harmless generic methods like this should be allowed in script-security
```

**Review by aHenryJard — APPROVED** [review]
Not a blocker, just a proposal on whitelist (but it's in comment anyway so not highly important)

**Review by aHenryJard — APPROVED** [review]

**Comment by Vlatombe** [comment]
@jglick Could you help us with merge/release of this one?

File: src/main/resources/hudson/matrix/Messages_da.properties [hunk 5]
```diff
@@ -23,7 +23,7 @@
 TextArea.DisplayName=Brugerdefineret akse
 MatrixProject.DuplicateAxisName=Dupliker akse navn
 MatrixBuild.Triggering=Starter {0}
-LabelAxis.DisplayName=Slaver
+LabelAxis.DisplayName=Agenter
```
Comment (jglick) [inline src/main/resources/hudson/matrix/Messages_da.properties:26]: Do you speak Danish? :grin:

Comment (twasyl) [reply src/main/resources/hudson/matrix/Messages_da.properties:26]: I have a good friend of mine who does, Google Translate. Maybe do you know him as well? ;)

Comment (jglick) [reply src/main/resources/hudson/matrix/Messages_da.properties:26]: LOL @javeleon thinking of you

**Pull request merged**